sfcc_cartridge/int_fast_sfra/cartridge/static/default/js/fastProduct.js

```diff
@@ -23,14 +47,13 @@ $(document).ready(function () {
                 
                 checkoutFastButton.on('click', function() {
                     if (isQuickViewModalOpen) {
-                    	quickviewQty = parseInt( $('#quickViewModal .quantity-select').find(":selected").val().trim() );
+                        quickviewQty = parseInt( $('#quickViewModal .quantity-select').find(":selected").val().trim() );
```

Comment:
would it be better/safer to add the `radix` in the `parseInt()` ?

Reply:
in parseInt() radix is Optional and Default value is 10. so not required to add radix here

Reply:
not always the case:

> Be careful—this does not default to 10

> If the input string begins with "0x" or "0X" (a zero, followed by lowercase or uppercase X), radix is assumed to be 16 and the rest of the string is parsed as a hexadecimal number.
If the input string begins with any other value, the radix is 10 (decimal).

so if the `string` is guaranteed to be non-`0x` prefixed I think it should be fine, otherwise the `radix` can be `16` in which case.

that's being said not very familiar with the business here, just suggesting :)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt